src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -498,11 +498,13 @@ private function getTemplateVarVouchers(Cart $cart)
 
             $vouchers[$cartVoucher['id_cart_rule']]['reduction_amount'] = $cartVoucher['reduction_amount'];
 
-            if (array_key_exists('gift_product', $cartVoucher) && $cartVoucher['gift_product']) {
+            if ((array_key_exists('gift_product', $cartVoucher) && $cartVoucher['gift_product']) ||
+                    $cartVoucher['free_shipping']) {
```

Comment:
I don't understand why we should have a `reduction_amount` in case of free shipping? :thinking:

Reply:
If the free delivery voucher used then the shipping value is the reduction amount.

Reply:
Could you please create an issue to check if our Product Team if everything's ok?

Reply:
https://github.com/PrestaShop/PrestaShop/issues/15460

Reply:
Thanks, I'm waiting for feedback from the issue before validating :)

Reply:
Since my issue is closed as duplicate, I'd like to prove link to the issue that exists for a long while https://github.com/PrestaShop/PrestaShop/issues/9702